packages/nns/candid/governance.certified.idl.js

```diff
@@ -629,21 +656,40 @@ export const init = ({ IDL }) => {
     'cf_neurons' : IDL.Vec(CfNeuron),
   });
   const Ballot = IDL.Record({ 'vote' : IDL.Int32, 'voting_power' : IDL.Nat64 });
+  const CanisterStatusResultV2 = IDL.Record({
+    'status' : IDL.Opt(IDL.Int32),
+    'freezing_threshold' : IDL.Opt(IDL.Nat64),
+    'controllers' : IDL.Vec(IDL.Principal),
+    'memory_size' : IDL.Opt(IDL.Nat64),
+    'cycles' : IDL.Opt(IDL.Nat64),
+    'idle_cycles_burned_per_day' : IDL.Opt(IDL.Nat64),
+    'module_hash' : IDL.Vec(IDL.Nat8),
+  });
+  const CanisterSummary = IDL.Record({
+    'status' : IDL.Opt(CanisterStatusResultV2),
```

Comment:
I know not related to your PR but, that's correct? Not sure what the `memory` and `cycles` balance of canister does in the governance proposal 🤷‍♂️

Reply:
Good question, I also don't understand why this is needed...